working_notebook-TO_COMPLETE_Anna.ipynb

```diff
@@ -815,19 +815,25 @@
   },
```

Comment:
<p>Same comment here as the previous one. Besides, a side note: name of parameters and name of variable may differ. This means that you can _pass in_ a <code>dct</code>    variable which maps to a <code>patients_data</code>  parameter in the function. </p><p> </p><p>That being said, exercise is very correct (semantically speaking): algorithm is correct. Just the code is a bit "convoluted" in variable names. </p><p> </p><p>A few suggestions on how to improve it: </p><ul><li>variable names. What does <code>tp</code>  meant, for example? why not just <code>patient</code>  for example? :)</li><li>In Python, there is a commodity function you can use to <code>sum</code>  values in a sequence. Have a look here: <a href="https://docs.python.org/3/library/functions.html#sum" target="_blank">Built-in functions: SUM</a></li><li>There is no need to initialise <code>av</code>  to zero.. and indeed there is no need for the <code>av</code>  variable in the first place ;)</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WebValley2021ReImagined/programming-for-data-science/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>